test/wildcard-matcher.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape')
```

Comment:
`'use strict'` ?

Reply:
_nod_ `use strict` at the top of every JS file should be the defacto choice.  I'm used to ESLint catching this -- maybe another bit of evidence to move away from `standard` or at least enhance our CI it with some additional ESLint rules

Reply:
👍